src/main/java/org/snomed/snowstorm/core/rf2/export/ExportService.java

```diff
@@ -235,15 +252,102 @@ private File exportRF2ArchiveFile(String branchPath, String filenameEffectiveDat
 			branchService.unlock(branchPath);
 		}
 	}
+	
+	private File exportReferenceSetsOnlyRF2ArchiveFile(String branchPath, String filenameEffectiveDate,
```

Comment:
Is it necessary to duplicate such a large portion of code? Putting some logic in the existing export method to skip concepts, descriptions and relationships if it's a refset only export would be a lot cleaner.
Cloning large methods means more maintenance and potential bugs in the future.